src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -65,12 +65,16 @@
         v-if="isParent"
         hidden
         :id="parentLabel"
-      >, containing {{ item.childUIDs.length }} symbols</span>
+      >, {{ $tc(
```

Comment:
Should we split these 3 up into 3 sentences? I think what we are doing here is similar to "string stitching".

Reply:
I don't understand what you mean by split these 3 up into 3 sentences. This is only one sentence but I'm passing the length of children to know if it's plural or not, and then the property for the number var in case it's plural

Reply:
I meant can we split this one sentence (that has 3 parts separated by comma) into three sentences to avoid stitching together sentences into 1 sentence. Or, we should consider just localize the entire sentence together.
Maybe the localized sentence flow differently if we localize the entire sentence vs stitching separate mini sentences together.

Reply:
I believe it's something we could improve in a future refactor for sure.

Reply:
@hqhhuang I created the radar rdar://105598877 for this refactor